README.md

```diff
@@ -1,6 +1,6 @@
 # Safe Core SDK Monorepo
 
-[![Logo](https://raw.githubusercontent.com/gnosis/safe-core-sdk/main/assets/logo.png)](https://gnosis-safe.io/)
+[![Logo](https://raw.githubusercontent.com/gnosis/safe-core-sdk/main/assets/Safe_Logos_Core_SDK_Green.png)](https://gnosis-safe.io/)
```

Comment:
Is `gnosis/safe-core-sdk` correct here?

Reply:
Good catch! It works meanwhile Gnosis does not create a new repo with the same name that overrites the route. I will update to `safe-global`.
Thanks!